.github/workflows/api-doc.yml

```diff
@@ -19,7 +19,7 @@ name: api-doc
 env:
   API_TYPES: 'OAS'
   API_DIRECTORIES: 'src/main/resources/swagger.api'
-  API_EXCLUDES: ''
+  API_EXCLUDES: 'src/main/resources/swagger.api/paths/authority-note-types/' 'src/main/resources/swagger.api/paths/authority-source-files/' 'src/main/resources/swagger.api/paths/authority-authority-storage/' 'src/main/resources/swagger.api/responses/'
```

Comment:
No! I provided explicit instructions about what needed to be done in api-doc.yml

See my [comment](https://github.com/folio-org/mod-entities-links/pull/127#issuecomment-1639065524).

Reply:
Done as ```API_EXCLUDES: 'paths responses'```